portsyncd/linksync.cpp

```diff
@@ -165,16 +165,18 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
     struct rtnl_link *link = (struct rtnl_link *)obj;
     string key = rtnl_link_get_name(link);
 
+    vector<FieldValueTuple> temp;
     if (key.compare(0, INTFS_PREFIX.length(), INTFS_PREFIX) &&
         key.compare(0, LAG_PREFIX.length(), LAG_PREFIX) &&
-        key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX))
+        key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX) &&
+        !m_portTable.get(key, temp))
```

Comment:
Have hardcoded prefix "Inband" instead of considering all the names from PORT table.

Reply:
Fixed